commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPAttachmentFileEntryLocalServiceImpl.java

```diff
@@ -203,7 +203,7 @@ public void checkCPAttachmentFileEntriesByDisplayDate(
 			serviceContext.setCommand(Constants.UPDATE);
 			serviceContext.setScopeGroupId(cpAttachmentFileEntry.getGroupId());
 
-			cpAttachmentFileEntryLocalService.updateStatus(
```

Comment:
@igorbeslic this way to call methods is necessary to support service wrapper extension points: https://portal.liferay.dev/docs/7-0/tutorials/-/knowledge_base/t/customizing-liferay-services-service-wrappers.  
Please revert this commit